app/controllers/feedback/post-index.controller.js

```diff
@@ -1,24 +1,20 @@
 'use strict'
 
 const lodash = require('lodash')
-
 const logger = require('../../utils/logger')(__filename)
 const paths = require('../../paths')
 const zendeskClient = require('../../services/clients/zendesk.client')
 
 module.exports = (req, res) => {
-  const message = `Service name: ${req.body['service-name']}
-External ID: ${req.body['service-external-id']}
-Gateway account: ${req.body['service-gateway']}
-Feedback rating: ${req.body['feedback-rating']}
+  const message = `Feedback rating: ${req.body['feedback-rating']}
 ----
 ${req.body['feedback-suggestion']}`
 
   const opts = {
     email: req.body.email,
     name: '(no name supplied)',
     type: 'question',
-    subject: `Feedback from: ${req.body['service-name']}`,
+    subject: `General feedback`,
```

Comment:
Shall we say we `Feedback from service` to make it clear for support?

Reply:
Done